config/initializers/geocoder.rb

```diff
@@ -2,6 +2,7 @@
   # geocoding options
   # :timeout      => 3,           # geocoding service timeout (secs)
   # :lookup       => :google,     # name of geocoding service (symbol)
+  :lookup => :test,
```

Comment:
You can run the app with 
`RAILS_ENV=production rails s` to simulate production locally